.circleci/main.yml

```diff
@@ -2164,6 +2095,47 @@ workflows:
             - python-sdk-test-publish
             - sign-off
           filters: *only-release-tags
+      - console-lint-backend:
+          filters: *only-release-tags      
+      - console-lint-frontend:
+          filters: *only-release-tags    
+      - console-lint-cypress:
+          filters: *only-release-tags   
+      - console-build:
```

Comment:
I don't think we need to run these for the release. Build is handled by the docker build. I was just mirroring what we have in the console workflow.